src/cloud/paths/tasks.yml

```diff
@@ -46,6 +46,20 @@ get:
         maximum: 500
         default: 100
       description: The non-zero number of tasks to return
+    - in: query
+      name: offset
+      schema:
+        type: integer
+        minimum: 0
+        default: 0
+      description: The number of records to skip
+    - in: query
+      name: sortBy
+      description: Field that records should be sorted by
+      schema:
+        type: string
+        enum:
+          - name
```

Comment:
can we mark these as not required so that the UI not sending them over won't change current behavior?

Reply:
@hoorayimhelping Yes, I will do that!